web/src/user_groups.ts

```diff
@@ -176,6 +176,15 @@ export function is_user_group(
     return item.members !== undefined;
 }
 
+export function is_empty_group(user_group_id: number): boolean {
+    const user_group = user_group_by_id_dict.get(user_group_id);
+    if (user_group === undefined) {
+        blueslip.error("Could not find user group", {user_group_id});
+        return false;
+    }
+    return user_group.members.size === 0 && user_group.direct_subgroup_ids.size === 0;
+}
```

Comment:
This is not a correct check for whether the group is empty; what if it contains a group that contains 0 members? Walking the subgroups a la `get_recursive_subgroups` is likely required to be correct.

Ideally, we should do this in a way that can `return true` as soon as it finds a member, so it runs in O(1) time, rather than counting all the members in a huge group just to confirm it's not empty.

Reply:
OK, I have updated the PR to check all the recursive subgroups.